tests/util/test_xsDateTime.py

```diff
@@ -0,0 +1,41 @@
+
+from datetime import datetime, timedelta
+from codalib.util import xsDateTime_parse, xsDateTime_format, XSDateTimezone
+
+
+def test_parse_date():
+
+    dt_str = "2017-01-27T14:43:00+00:00"
+
+    assert isinstance(xsDateTime_parse(dt_str), datetime)
+    dt = xsDateTime_parse(dt_str)
```

Comment:
Why not reverse these two so you only call the `xsDateTime_parse` function once then verify that it returns a datetime?